molecule/certificates.yml

```diff
@@ -109,6 +134,20 @@
           loop: "{{groups['ldap_server']}}"
           when: groups['ldap_server'] is defined
 
+    - name: Install bind utils
+      yum:
+        name:
+          - bind-utils
+        state: present
+      when: ansible_os_family == "RedHat"
+
+    - name: Install bind utils
+      apt:
+        name:
+          - dnsutils
+        state: present
+      when: ansible_os_family == "Debian"
```

Comment:
Where are we using it? Whats the recommendation for air-gapped setups

Reply:
we're using this because FIPS needs to have IP address in SAN. [command](https://github.com/confluentinc/cp-ansible/blob/2c8117fad8d7e1f31747bd742ef0e1d0f0d66002/molecule/certs-create.sh#L53) to get IP address uses dig. This is only used in our ldap test setup so customers would set this up on their own. Since the task is part of molecule, it does not affect air-gapped installation